Aimec doc (#741)
avaframe/AvaFrame: docs/moduleAna3AIMEC.rst

*  avalanche path in LINES (as a shapefile named ``path_aimec.shp``)
*  a splitPoint in POINTS (as a shapefile named ``splitPoint.shp``)
*  Results from avalanche simulation (when using results from com1DFA,
   the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in
   :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
* a method to define the reference simulation. By default, an arbitrary simulation

>   is defined as reference (this can be changed in the ``ana3AIMEC/local_ana3AIMECCfg.ini``
>   as explained in :ref:`moduleAna3AIMEC:Defining the reference simulation`)

Reviewer: ```suggestion
  is defined as reference. This can be changed in the ``ana3AIMEC/local_ana3AIMECCfg.ini``
  as explained in :ref:`moduleAna3AIMEC:Defining the reference simulation`.
```

.. Note:: The spatial resolution of the DEM and its extend can differ from the result raster data.
          Spatial resolution can also differ between simulations. If this is the case, the spatial
          resolution of the reference simulation results rasters is used (default) or the resolution
          specified in the configuration file (``cellSizeSL``) is used if this one is provided.

>           This way, all simulations will be transformed and analyzed using on single resolution.

Reviewer: ```suggestion
          This way, all simulations will be transformed and analyzed using the same resolution.
```

projected into the curvilinear system using the previously described method. The maximum and
average values of those fields are computed in each cross-section (l direction).
For example the maximum :math:`A_{cross}^{max}(s)` and average :math:`\bar{A}_{cross}(s)` of
the two dimensional distribution :math:`A(s,l)` is:

.. math::
    A_{cross}^{max}(s) = \max_{\forall l \in [-\frac{w}{2},\frac{w}{2}]} A(s,l) \quad\mbox{and}\quad
    \bar{A}_{cross}(s) = \frac{1}{w}\int_{-\frac{w}{2}}^{\frac{w}{2}} A(s,l)dl

Runout point
~~~~~~~~~~~~~

The runout point is always given in regards to a peak result field (:math:`A(s,l)` which could be
peak pressure of flow thickness...) and a threshold value (:math:`A_{lim}>0`).
The runout point (:math:`s=s_{runout}`) and the related :math:`(x_{runout},y_{runout})`
in the original coordinate system, corresponds to the last point in flow direction where the
chosen peak result :math:`A_{cross}^{max}(s)` is above the threshold value :math:`A_{lim}`.

.. Note:: It is very important to note that the position of the runout point depends on the chosen
          threshold value and peak result field. It would also be possible to use

>           :math:`\bar{A}_{cross}(s)>A_{lim}` instead of :math:`A_{cross}^{max}(s)>A_{lim}`

Reviewer: Would 'It is also possible' describe it better than 'it would also be possible'?

Author: Yes we can use it. Then I kind of understand that there is an option to change this somewhere in the ini files...

This section describes how the theory is implemented in the ``ana3AIMEC`` module.

Defining the reference simulation
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In order to fully proceed to the Aimec analysis, a reference simulation needs to be defined.
> The analysis of the other simulations  will then be compared to the one of the reference simulation.
> The reference simulation can be defined by its name (or part of the name) or based on some
> configuration parameter and value if it comes from the :py:mod:`com1DFA` module
> (or any computational module that provides a configuration):

Reviewer: ```suggestion
To apply a complete Aimec analysis, a reference simulation needs to be defined.
The analysis of the other simulations will be compared to the one of the reference simulation.
The reference simulation can be determined by its name (or part of the name) or based on some
configuration parameter and value if it comes from the :py:mod:`com1DFA` module
(or any computational module that provides a configuration):
```

  * Based on the simulation name

>      one needs to provide a not empty string in the AIMEC configuration

Reviewer: ```suggestion
     one needs to provide a non-empty string in the AIMEC configuration
```
